drivers/hmis/app/graphql/types/hmis_schema/has_projects.rb

```diff
@@ -9,6 +9,7 @@
 module Types
   module HmisSchema
     module HasProjects
+      include ArelHelper
```

Comment:
We can probably drop the `include ArelHelper` here if the sorting logic is moved to the model